snack-shop-bff/src/snacks/snacks.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller('snacks')
+export class SnacksController {
+  @Get()
+  findAll() {
+    const snacks = [
```

Comment:
@dowenb As we just aligned on a revised model, could you please update the mock data here accordingly? Thanks!

Reply:
@lisihocke done, I've also implemented an /orders endpoint with an example return value. Please review.

Reply:
![image](https://github.com/dowenb/snack-shop/assets/33220797/df509e62-6214-4f60-b367-cd8473c91643)

Reply:
![image](https://github.com/dowenb/snack-shop/assets/33220797/be69ad10-0ac0-4a0f-879f-fa368402683a)